Domain.Sql/Migrations/AzureSqlDbMigrator.cs

```diff
@@ -94,12 +95,36 @@ public MigrationResult Migrate(DbContext context)
             var sql =
                 $@"
 alter database {context.Database.Connection.Database} 
-modify (MAXSIZE = {MaxSize},
+modify (MAXSIZE = {MaxSize}MB,
```

Comment:
What about when the max size is in GB? Do all increments work or does Azure SQL Database understand only specific values? I'm wary about doing any math or validation as opposed to treating this as a string.